### DISCUSSION TITLE
"Remember me" functionality

### DISCUSSION THREAD

**Discussion by mvoud-bel** [simpleidserver/SimpleIdServer#651] [Q&A]
Hello,
Correct me if I am wrong but I believe there is an issue on the "Remember me" functionality.

**Repro Steps:**
1.Click Authenticate 
2.Sign in with "Remember me" checked.
3.Close and re-open the browser to https://localhost:5001/master .
Result: My session isn't kept and all cookies are deleted.

**Expected Result:**
Keep me signed in until I sign out, delete cookies from browser or cookie expiration date is passed.

**Comment by simpleidserver** [comment]
Hello :)

Indeed, there was an issue on the IdServer website where the cookie parameters were overridden by the OpenIdConnectHandler.
The problem has now been resolved in the `master` branch (Ticket #652).

KR,

SID
